lib/wallet.js

```diff
@@ -77,17 +75,8 @@ function sendCoins (settings, toAddress, cryptoAtoms, cryptoCode) {
 }
 
 function newAddress (settings, info) {
-  const walletAddressPromise = fetchWallet(settings, info.cryptoCode)
+  return fetchWallet(settings, info.cryptoCode)
```

Comment:
Need to update fetchWallet so that it returns the layer2 plugin, when enabled.

Reply:
Instead of wallet? If I do that, I will have to write second `fetchWallet` and `newAddress` for only wallet without layer2 because `layer2.newAddress` uses `wallet.newAddress`, so wallet fetching will be executed twice. 
Did I get your review?

Reply:
Now I think i got it, I can make composite of wallet and layer2.